### DISCUSSION TITLE
Timezone change from UTC to EST or CST and also time format 24hrs to AM/PM

### DISCUSSION THREAD

**Discussion by dmytrokoren** [jdholtz/auto-southwest-check-in#227] [General]
Is it possible to add feature to change the timezone and time format in the notification message:

As Example right now it shows:
Successfully scheduled the following flights to check in for ...:
Flight from New York (LaGuardia) to Dallas (Love Field) at **2024-02-10 18:40:00 UTC**

Would it be possible to set the timezone and time format in config.json like so,
"timezone": "CST",
"time_format": "24hrs" or "AM/PM"

and it would notify as:
Successfully scheduled the following flights to check in for ...:
Flight from New York (LaGuardia) to Dallas (Love Field) at **2024-02-10 05:45:00 PM CST**

timezone feature is more important for me as that would tell me when my check will occur, not so much time_format but it would be nice.

**Comment by jdholtz** [comment]
Yes, this would be a good feature. It’s probably just best to send the times as local to the flight (e.g. EST if New York, CST if Chicago, etc.) instead of sending as UTC. 

Therefore, there wouldn’t be a config option for timezone, but still a config option for 12/24 hour format. Thoughts?

**Comment by dmytrokoren** [comment]
> Yes, this would be a good feature. It’s probably just best to send the times as local to the flight (e.g. EST if New York, CST if Chicago, etc.) instead of sending as UTC. 

I agree to send out in local time zone. No configuration values required. But to make sure it displays EST or CST so there won't be any confusion.
> 
> Therefore, there wouldn’t be a config option for timezone, but still a config option for 12/24 hour format. Thoughts?

Yes just configuration for "24-Hour Time" = false by default

If someone wants to be 24 hour format then they will have to set as true.

What you think?

**Reply by jdholtz** [reply]
Yeah, since most people prefer 12 hours and it is the default in almost all software (in America at least), then 12-hour time can be the default.

**Comment by jdholtz** [comment]
Added in the develop branch and will be in the next release.
